examples/fluids/blasius.yaml

```
@@ -4,7 +4,7 @@ implicit: true
 ts:
   adapt_type: 'none'
   type: 'beuler'
-  dt: 5.e-8
+  dt: 6.4e-5
```

Can we set a max time? I think this is about 10 velocity transit times through the domain, and the initial SNES residuals are really small by this time.
```suggestion
  dt: 6.4e-5
  max_time: 1e-3
```

---

Yes.  This is needed.